backend/lib/rpc-service/service.js

```diff
@@ -0,0 +1,91 @@
+import { RPC_URL } from '../config.js'
+import { base64pad } from 'multiformats/bases/base64'
+import { encode as cborEncode } from '@ipld/dag-cbor'
+import { decode as jsonDecode } from '@ipld/dag-json'
+import { request } from 'undici'
```

Comment:
Let's use the built-in Fetch API instead of undici. It's always better to have less dependencies. Unless you need a specific feature that undici provides but the built-in API does not?